src/components/Choice/src/Choice.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+	<div
+		:class="[
+			$s.Container,
+		]"
+	>
+		<slot />
+	</div>
+</template>
+
+<script>
+import assert from '@square/maker/utils/assert';
+import key from './key';
+
+export default {
+	provide() {
+		return {
+			[key]: this.$data,
+		};
+	},
+
+	model: {
+		prop: 'selected',
+		event: 'choice:update',
+	},
+
+	props: {
+		/**
+		 * Selected choice option
+		 */
+		selected: {
+			type: undefined,
```

Comment:
Is there a reason we want to leave this `undefined`?

Reply:
I believe this is because `selected` is either a `String` in `single-select` mode and an `Array` in `multi-select` mode. Although `undefined` works here I would try `[String, Array]` because I believe Vue also allows passing an array of constructors in the `type` field of the prop validation objects.

Reply:
@pretzelhammer agreed, I think `[String, Array]` makes more sense here

Reply:
Oh, @sohnj877 intent here might be to allow _any_ value in `single-select` mode, not just `String`s but also `Number`s, `Object`s, etc hence why the type is `undefined`.

Reply:
My intention was to allow any types.